src/lib/features.json

```diff
@@ -2,131 +2,136 @@
   {
     "name": "unloadChunkByEmptyChunk",
     "description": "Chunk unloading is done by sending an empty chunk",
-    "versions": ["1.8"]
+    "versions": ["1.8.x"]
   },
   {
     "name": "unloadChunkDirect",
     "description": "Chunk unloading is done by sending directly an unload chunk packet",
-    "versions": ["1.9", "1.10", "1.11", "1.12", "1.13", "1.14", "1.15", "1.16"]
+    "versions": ["1.9.x", "1.10.x", "1.11.x", "1.12.x", "1.13.x", "1.14.x", "1.15.x", "1.16.x"]
   },
   {
     "name": "fixedPointPosition",
     "description": "Entity positions are represented with fixed point numbers",
-    "versions": ["1.8"]
+    "versions": ["1.8.x"]
```

Comment:
Could you please instead follow the format used in mcdata and copy the support feature implementation from there

Reply:
The format is this `1.10_major`? if yes then
1.10_major = 1.10.x

and latest would be what? the latest version?